lm-cli/lm_cli/constants.py

```diff
@@ -0,0 +1,15 @@
+"""
+Provide constants that may be used throughout the CLI modules.
+"""
+
+from enum import Enum
+
+
+class SortOrder(str, Enum):
+    """
+    Enum descring the type of sort orders that are available for list commands.
+    """
+
+    ASCENDING = "ascending"
+    DESCENDING = "descending"
+    UNSORTED = "unsorted"
```

Comment:
If you agree, I suggest:

```suggestion
    ASCENDING = "ASCENDING"
    DESCENDING = "DESCENDING"
    UNSORTED = "UNSORTED"
```

Reply:
I decided to put this in lowercase so the user doesn't have to pass the sort order in uppercase, since it would be the only command in uppercase...